Fixed Ending Screen Buttons

joeboby opened the pull request:
Changed the buttons to match those on the main menu

Relates to issue #99

joeboby committed:
Fixed Ending Screen Buttons

Changed the buttons to match those on the main menu

joeboby committed:
Basics of Customization Menu

Started working on the customization menu have a selection screen and started making a class to hold the skins

joeboby committed:
Saving State

Added the basics for saving skin data (not currently tested or used)

penner75 reviewed (commented):
Looks good, just need to move the conflicting LoadScene in BirdController to where the other LoadScene currently is (near the beginning of update)

joeboby committed:
Saving State

Continued working on saving game state so that it persists between loading. I think it is done now but needs to be tested.

Relates to #112 #113 #114 #116

joeboby committed:
Saving and Loading bug fixes

Working on loading and saving to make sure they work, currently not reading in the image file after loading in the privative data

BrittBarber committed:
Merge branch 'dev' into JohnsDev3

BrittBarber reviewed (approved):
Looks good
